Makefile

```diff
@@ -34,7 +35,6 @@ deb: build
 	-t deb \
 	-v $(VERSION) \
 	--deb-no-default-config-files \
-	--deb-use-file-permissions \
```

Comment:
not really lucky. ironically we still get the same permissions we set (tested with different permissions). I see `--deb-user USER` and `--deb-group GROUP` per https://github.com/jordansissel/fpm/wiki, but

```
...
--deb-user root \
--deb-group root \
...
```

seem to have no effect with `--deb-use-file-permissions` (files still owned by ubuntu:ubuntu after installation)